### PR TITLE
fix: update preferences in default Firefox profile

### PR DESCRIPTION
**Pull request by mjzffr**

### CODE REVIEW

**Review by hanselfmu — APPROVED** [review]

**Comment by hanselfmu** [comment]
Hi Maja, feel free to squash and merge this in anytime when you see fit. Thanks!

**Comment by mjzffr** [comment]
Thanks @hanselfmu 
I don't have write access, so someone else will have to merge.

**Review by mathiasbynens — APPROVED** [review]

**Pull request merged**
